tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -55,6 +56,78 @@ def check_interface_status(duthost):
     return False
 
 
+def query_counter_result(duthost, query_key):
+    counters_query_string = 'sonic-db-cli COUNTERS_DB hgetall "DHCPV4_COUNTER_TABLE:{key}"'
+    result = json.loads(
+        duthost.shell(counters_query_string.format(key=query_key))['stdout'].replace("\"", "").replace("'", "\"")
+    )
+    return {k: {ik: int(iv) for ik, iv in v.items()} for k, v in result.items()}
```

Comment:
We can use meaningful naming like
{dir: {pkt_type: int(count) for pkt_type, count in counters.items()} for dir, counters in result.items()}